### ISSUE TITLE
Not working on Hulu

### ISSUE THREAD

**Issue by bradlymathews** [dvlden/ultrawideo#16]
I see only the lower right quadrant of the video in the upper left quadrant of the screen. I have tried all modes. Latest Chrome browser on Windows 10 on Dell U3818DW. Works fine in HBO Now, Netflix and Youtube.

**Comment by dvlden** [comment]
I don't have access to the Hulu. If you are willing to provide me the access via e-mail to investigate and change your PW afterward, I'd be happy to look into it. `nenad@dvl.sh`

**Comment by OPAISDAYDREAMING** [comment]
Maybe, you could set up an account for a free trial if people are unwilling to give their account info.

**Comment by dvlden** [comment]
Yes, thanks. I tried that, but apparently, it doesn't like my credit card or country.
http://drop.dvl.sh/aCN3Ln

I can try with VPN and the same card, but if it fails, I'd really appreciate if someone is willing to share, until I make a fix for this extension.

Come on people, it's open-source. If it's not working, it's the least you can do to help me out. 😞

**Comment by dvlden** [comment]
It's sad that people don't want to help with such a small thing... _(account credentials)_
I was able to purchase a Hulu account _(it's the USA only apparently)_ and to debug to find out the issue.

The problem is not so much with the extension, but instead with a Hulu platform.

`document.fullscreenElement` returns the `<div />` container element which has three different video elements inside.

<img width="433" alt="Screen Shot 2019-08-06 at 10 00 31 AM" src="https://user-images.githubusercontent.com/3460368/62522146-5d50de80-b831-11e9-867d-1657d923353f.png">

The last one is the actual fullscreen video that we're looking at.
In order to fix this problem, all I can do is make this platform-specific workaround, where I'd check if a fullscreen element contains `hulu-player-app` class and if the condition matches, I can find all video players within and grab the last one.

The problem with such a solution is that it's bound to the Hulu platform only.

The other approach is to always get all video elements and always just take the last one, but that's not the case with other platforms so it'd most likely break on Twitch platform _(it actually won't, I just checked)_.

Well, I'm unsure. Will consider what is the best approach and provide it.
